client/project.go

```diff
@@ -24,8 +24,13 @@ func (self *ApiClient) Project(id string) (Project, error) {
 
 func (self *ApiClient) ProjectCreate(name string) (Project, error) {
 	var result Project
-	request := map[string]interface{}{"name": name}
-	err := self.http.Post("/projects", request, &result)
+	organizationId, err := self.organizationId()
+	if err != nil {
+		return Project{}, nil
```

Comment:
This shouldn't be - 
```suggestion
		return Project{}, err
```
?

Reply:
Yes it should thanks!